database/migrations/2016_05_02_173914_move_featured_reportback_to_competition.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class MoveFeaturedReportbackToCompetition extends Migration
```

Comment:
@angaither just chatted with @deadlybutter IRL. He's gonna remove some of these migrations that haven't been pushed to prod yet and were only meant to fix migrations just run locally :) Cleaning them up and simplifying 💃